MachOKit/DataInCode/MKDataInCode.m

```diff
@@ -113,16 +113,9 @@ - (void)dealloc
 //|++++++++++++++++++++++++++++++++++++|//
 - (MKOptional*)childNodeOccupyingVMAddress:(mk_vm_address_t)address targetClass:(Class)targetClass
 {
-    for (MKDataInCodeEntry *entry in self.entries) {
-        mk_vm_range_t range = mk_vm_range_make(entry.nodeVMAddress, entry.nodeSize);
-        if (mk_vm_range_contains_address(range, 0, address) == MK_ESUCCESS) {
-            MKOptional *child = [entry childNodeOccupyingVMAddress:address targetClass:targetClass];
-            if (child.value)
-                return child;
-            // else, fallthrough and call the super's implementation.
-            // The caller may actually be looking for *this* node.
-        }
-    }
+    MKOptional *child = [MKBackedNode childNodeOccupyingVMAddress:address targetClass:targetClass inSortedArray:(NSArray *)self.entries];
```

Comment:
This creates an implicit requirement that `entries` is sorted in address order.  A unit test to verify this should be added, to prevent subtle bugs from later slipping in.  Same for all the other nodes with the change.

I can handle that after the PR is merged.